File: src/test/java/de/akquinet/ccsp/floodwatch/datamodel/LocationMother.java
package de.akquinet.ccsp.floodwatch.datamodel;

import org.springframework.data.mongodb.core.geo.GeoJsonPoint;

public final class LocationMother {

    private LocationMother() {
    }

    public static Location create() {
        Location location = new Location("5479", new GeoJsonPoint(46.045512f, 14.451169f), "Gradaščica", "Bokalce", "Gradaščica - Bokalce");
        location.setAltitude(296.63f);
        return location;
    }
}

File: src/test/java/de/akquinet/ccsp/floodwatch/datamodel/LocationMother.kt
package de.akquinet.ccsp.floodwatch.datamodel

import org.springframework.data.mongodb.core.geo.GeoJsonPoint

object LocationMother {

    fun create(): Location {
        val location = Location("5479", GeoJsonPoint(46.045512, 14.451169), "Gradaščica", "Bokalce", "Gradaščica - Bokalce")
        location.altitude = 296.63f
        return location
    }
}